torch/_tensor.py

```diff
@@ -506,6 +506,12 @@ def split(self, split_size, dim=0):
         else:
             return super(Tensor, self).split_with_sizes(split_size, dim)
 
+    def pad(self, pad_width, mode='constant', *, constant_values=None, out=None):
```

Comment:
Would you just add a comment explaining why registering this method in Python is necessary?